tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -1050,6 +1050,19 @@ def find_interface(tests, id=nil, skipcheck=true)
     msg = 'Unable to find suitable interface module for this test.'
     prereq_skip(tests[:resource_name], self, msg)
   end
+
+  if type =~ /ethernet/i
```

Comment:
* This block should be part of the ethernet case block above. 
* This block should only process for modular switches, `if platform[/n7k/]`
* Tests that use feature vni will fail with 'f2e f3', must use 'f3' only
* We have other methods already that try to handle f3 setup: `setup_mt_full_env`, `setup_fabricpath_env`, etc. How will this change play with those methods? I think we may want to take a step back and come up with a comprehensive setup instead.